### DISCUSSION TITLE
Pi zero enough power?

### DISCUSSION THREAD

**Discussion by RaffoNin** [Koenkk/zigbee2mqtt#12345] [Devices]
I'm thinking of starting a new Z2M network on the second floor of my house and was wondering if a pi zero is enough. 

I only plan on running Z2M. 

Note: my house has a lot of thick concrete that makes it impossible to have a proper mesh network between floors.

**Comment by deviantintegral** [comment]
You may be interested in https://github.com/Koenkk/zigbee2mqtt/discussions/13599 - I think if you don't care about mDNS, the Pi Zero may only be OK if you're using docker and not having to run `npm ci` on it directly.
